test/framework/github.py

```diff
@@ -551,6 +552,40 @@ def test_det_patch_specs(self):
         self.assertEqual(os.path.basename(res[2][0]), '3.patch')
         self.assertEqual(res[2][1], 'C')
 
+    def test_restclient(self):
+        """Test use of RestClient."""
+        if self.skip_github_tests:
+            print("Skipping test_restclient, no GitHub token available?")
+            return
+
+        client = RestClient('https://api.github.com', username=GITHUB_TEST_ACCOUNT, token=self.github_token)
+
+        status, body = client.repos['hpcugent']['testrepository'].contents.a_directory['a_file.txt'].get()
+        self.assertEqual(status, 200)
+        # dGhpcyBpcyBhIGxpbmUgb2YgdGV4dAo= == 'this is a line of text' in base64 encoding
```

Comment:
why don't you encode this on the fly instead of this comment?

Reply:
I stole this from the `vsc-base` checks, I have to admit I don't know why it was done like this.

Will try and encode on the fly...

Reply:
fixed in e0e1e16ab